muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/coursepicker/CoursePickerRESTService.java

```diff
@@ -581,7 +617,28 @@ private boolean getCanSignup(WorkspaceEntity workspaceEntity) {
     }
   }
   
-  private CoursePickerWorkspace createRestModel(WorkspaceEntity workspaceEntity, String name, String nameExtension, String description, String educationTypeName, boolean canSignup, boolean isCourseMember) {
+  private boolean getIsAlreadyEvaluated(WorkspaceEntity workspaceEntity) {
+    if (sessionController.isLoggedIn()) {
+      WorkspaceUserEntity workspaceUserEntity = workspaceUserEntityController.findWorkspaceUserByWorkspaceEntityAndUserIdentifier(workspaceEntity, sessionController.getLoggedUserEntity().defaultSchoolDataIdentifier());
+      Boolean isUnassessed = false;
```

Comment:
Reverse the logic and rename this variable to `isEvaluated` because it's highly misleading that the method name is `getIsAlreadyEvaluated` but its return value is actually the complete opposite.